app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/GameActivity.java

```diff
@@ -2,12 +2,45 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.Button;
 
-public class GameActivity extends AppCompatActivity {
+import nl.tudelft.pixelperfect.client.EventCompletedMessage;
+import nl.tudelft.pixelperfect.client.GameClient;
+import nl.tudelft.pixelperfect.event.AsteroidFieldEvent;
 
+/**
+ * The GameActivity page is a page where game starts. It is the main menu of the Android app.
+ */
+public class GameActivity extends AppCompatActivity implements View.OnClickListener {
+    private GameClient game;
+
+    /**
+     * This method shows what happens when this Activity is created.
+     *
+     * @param savedInstanceState , a Bundle.
+     */
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_game);
+
+        Button button = (Button) findViewById(R.id.button_complete_event);
+        button.setOnClickListener(this);
```

Comment:
This implementation will not work if we have multiple buttons. You can specify the method that must be called in the activity for each button in the XML. I suggest we use this approach. (Take a look at the main activity and its XML sheet, it is done there.)